### PR TITLE
Create weet

### PR DESCRIPTION
**Pull request by dsandoval11**
## Summary
- Add endpoint for weet creation

## Trello Card
- https://trello.com/c/GcsSXX72/33-crear-un-weet

### CODE REVIEW

File: app/helpers/number.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const getRandomInt = (min, max) => Math.floor(Math.random() * (max - min)) + min;
+
+module.exports = {
+  MIN: 1,
```
Comment (marioportela1993) [inline app/helpers/number.js]: Put this value in the constants file

File: app/helpers/number.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  MAX: 1000,
```
Comment (marioportela1993) [inline app/helpers/number.js]: Same comment

File: app/routes.js [hunk 3]
```diff
@@ -17,4 +18,6 @@ exports.init = app => {
     [validateBody(signupSchema), validateToken, validateRole(ADMIN)],
     users.signupAdmin
   );
+
+  app.post('/weets', [validateToken], weets.createWeet);
```
Comment (marioportela1993) [inline app/routes.js:22]: Validate please the data sent in the body

**Review by marioportela1993 — APPROVED** [review]
Nice Job, apply comments and then merge the PR :)

File: app/routes.js (hunk 3 above)
Comment (dsandoval11) [reply app/routes.js:22]: the endpoint doesn't receive data in the body 😅

**Pull request merged**